funnel/templates/account_merge.html.jinja2

```diff
@@ -4,21 +4,21 @@
 
 {% macro accountinfo(user) %}
   <ul class="mui-list--aligned mui--text-subhead">
-    <li><strong>{% trans name=user.fullname %}Name:</strong> {{ name }}{% endtrans %}</li>
-    <li><strong>{% trans username=user.username or '' %}Username:</strong> {{ username }}{% endtrans %}</li>
+    <li><span class="mui--text-bold">{% trans %}Name:{% endtrans %}</span> {{ user.fullname }}</li>
+    <li><span class="mui--text-bold">{% trans %}Username:{% endtrans %}</span> {{ user.username or '' }}</li>
```

Comment:
Should be `or gettext('(none)')`

Reply:
Better: `{% if user.username %}{{ user.username }}{% else %}{% trans %}(none){% endtrans %}{% endif %}`